charts/opentelemetry-demo/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.30.3
+version: 0.30.5
```

Comment:
this should be `0.30.4`

Reply:
I set it to 0.30.5 in anticipation of #1156 getting merged.